src/db.ts

```diff
@@ -24,10 +27,23 @@ export const client = <
 		throw new Error('Missing required environment variable: DB_CONNECTION_STRING');
 	}
 
+	const doLog = c.env.DB_ENABLE_LOGGING === 'true' ? true : false;
```

Comment:
Instead of adding an environment variable for this, can we expose the full `DrizzleConfig` as a parameter for this function?